docs/guides/sdk/getting-started.md

```diff
@@ -0,0 +1,250 @@
+};
+
+const client = useDirectus<Schema>('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(readItem('articles', 5));
+```
+
+#### Read all items
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(readItems('articles'));
+```
+
+#### Read specific fields
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(
+	readItems('articles', {
+		fields: ['id', 'title'],
+	})
+);
+```
+
+#### Read all fields
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(
+	readItems('articles', {
+		fields: ['*'],
+	})
+);
+```
+
+#### Read nested fields
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(
+	readItems('articles', {
+		fields: ['*', { author: ['*'] }],
+	})
+);
+```
+
+### Using GraphQL
+
+Add the `graphql()` composable to the client, this enables the `.query(...)` method to query the collection.
+
+For example, to make a request to an `articles` collection.
+
+```ts
+import { useDirectus } from '@directus/sdk@beta';
+import { graphql } from '@directus/sdk@beta/graphql';
+
+type Article = {
+	title: string;
+	content: string;
+};
+
+type Schema = {
+	articles: Article;
+};
+
+const client = useDirectus<Schema>('http://directus.example.com')
+	.use(graphql());
+
+const result = await client.query<Article[]>(`
+    query {
+        articles {
+            id
+            title
+            content
+        }
+    }
+`);
+```
+
+> Feel free to customize the queries according to your project's GraphQL schema.
+
+## Next Steps
+
+You have successfully created a composable client using the Directus SDK.
+
+As you progress, you may want to explore other available features like real-time updates, and GraphQL subscriptions.
+
+Keep an eye on our documentation as we continue to enhance and expand the SDK.
+
+If you encounter any issues or have questions, don't hesitate to reach out to us in our
+[Discord community](https://directus.chat/).
+
+## Older SDK Version
+
+The older SDK still works and your existing applications that use the older version will continue to
+function without any issues.
```

Comment:
Should we maybe just drop this? It feels somewhat misleading as `"existing applications that use the older version will continue to function without any issues."` only holds true if they have pinned the dependency to their old version

Reply:
That makes sense, I'll take it out.